src/map/lua/lua_baseentity.cpp

```diff
@@ -13187,7 +13188,7 @@ bool CLuaBaseEntity::getClaimable()
 {
     if (auto* PMob = dynamic_cast<CMobEntity*>(m_PBaseEntity))
     {
-        return PMob->m_IsClaimable;
+        return PMob->getMobMod(MOBMOD_CLAIM) < ClaimType::Unclaimable;
```

Comment:
Check for the two claimable types, rather than `< Unclaimable` (will work in line with enum class)